### PR TITLE
zTPF_OMR_Port_Pull2

### LINKED ISSUE #2: Missing .gitignore for generated artifacts
After I build the standalone OMR I ran "git status" and it showed lots of new files.  We missed the .gitignore in the initial commit.

### LINKED ISSUE #3: omrsig fails to build against glibc 2.21 or newer

### PR DESCRIPTION
**Pull request by jjohnst-us**
Description:
This pull request is the second in a series of committing omr changes
for the z/TPF platform. This pull request is a follow-on to PR #1147.
A successful personal build was done in the IBM environment
(build_id=351866), there were two or three compilation failures that
were fixed.  Unlike PR #1147 there are a small amount of platform
common files which were updated.

It's expected there will be two more pull requests after this one of
smaller sizes then followed by pull requests for specific fixes as part
of z/TPF builds and runtime fixes.

The current z/TPF build procedures are still a work-in-progress.

Most of the formatting was pre-existing.  Some of the files maybe
removed once our build is finalized.  So no need to overly analyze this
pull requests as we still have passes coming as we work out our build
and runtime bugs.

Signed-off-by: jjohnst-us <jjohnst@us.ibm.com>

### CODE REVIEW

**Comment by jjohnst-us** [comment]
@charliegracie *ping*

File: port/unix/omrfile.c [hunk 1]
```diff
@@ -353,7 +356,7 @@ omrfile_open(struct OMRPortLibrary *portLibrary, const char *path, int32_t flags
 #endif
 
 	if (-1 == fd) {
-		Trc_PRT_file_open_Exception2(path, errno, findError(errno));
```
Comment (charliegracie) [inline port/unix/omrfile.c]: Did you mean to remove the function call? This is common code and the change will have an impact on other platforms will it not?

**Comment by charliegracie** [comment]
Also could you update your commit message to be more similar to this with the lines of the body of the commit message wrapping at 72 characters as my formatting below does not do that :).  This way the commit message has the details so that it is easier to find in the future.  Also you only need one signed-off tag. Thanks.
```
Additional zTPF support

This pull request is the second in a series of committing omr changes for the z/TPF platform. This pull request is a follow-on to PR #1147 . A successful personal build was done in the IBM environment (build_id=351866), there were two or three compilation failures that were fixed. Unlike PR #1147 there are a small amount of platform common files which were updated.

It's expected there will be two more pull requests after this one of smaller sizes then followed by pull requests for specific fixes as part of z/TPF builds and runtime fixes.

The current z/TPF build procedures are still a work-in-progress.

Signed-off-by: jjohnst-us <jjohnst@us.ibm.com>
```

**Comment by jjohnst-us** [comment]
Re-formatted commit message per your request.

**Comment by jjohnst-us** [comment]
All the segments of this Pull were copied or modified from within OMR itself.

File: port/unix/omrfile.c (hunk 1 above)
Comment (jjohnst-us) [inline port/unix/omrfile.c]: No, didn't mean to remove it, nice catch.  Yes common code updates in this pull would affect other platforms, which is why I did a personal VM build.  The segments under z/TPF directory shouldn't affect other platforms.

**Comment by charliegracie** [comment]
@jjohnst-us you updated the PR comment and not the commit message.  Can you update the commit message and push it again please?  Thanks

**Comment by jjohnst-us** [comment]
@charliegracie Update made.  Not sure how to prevent the check builds from kicking off when just doing a commit message change.

Let me know if you need anything else.

**Pull request merged**